private/bufpkg/bufmodule/module.go

```diff
@@ -39,6 +40,8 @@ type module struct {
 	license              string
 	breakingConfig       *bufbreakingconfig.Config
 	lintConfig           *buflintconfig.Config
+	manifest             manifest.Manifest
+	blobs                manifest.BlobSet
```

Comment:
This is another data source. We should calculate these based on the sourceReadBucket, or viceversa, calculate the bucket based on the m+b.

Reply:
If modules are immutable once created, I think there's no issue with having the data in both formats (bucket and m+b) to avoid re-calculating every time. I guess my real concern is being able to pass 2 sources at the constructor.

Reply:
Fixed.